apps/transport/lib/jobs/dataset_quality_score.ex

```diff
@@ -412,49 +421,38 @@ defmodule Transport.Jobs.DatasetFreshnessScore do
             :metadata_inserted_at => binary | nil
           }
   def resource_freshness(%DB.Resource{format: "GTFS" = format, id: resource_id}) do
-    resource_history = resource_id |> DB.ResourceHistory.latest_resource_history()
-
-    if !is_nil(resource_history) and DB.ResourceHistory.gtfs_flex?(resource_history) do
-      %{
-        freshness: 1.0,
-        raw_measure: %{source: "gtfs_flex"},
-        metadata_id: nil,
-        metadata_inserted_at: nil
-      }
-    else
-      resource_id
-      |> DB.MultiValidation.resource_latest_validation(Transport.Validators.GTFSTransport)
-      |> case do
-        %{
-          metadata: %{
-            metadata: %{"start_date" => start_date, "end_date" => end_date},
-            id: metadata_id,
-            inserted_at: inserted_at
-          }
+    (DB.MultiValidation.resource_latest_validation(resource_id, Transport.Validators.GTFSTransport) ||
+       DB.MultiValidation.resource_latest_validation(resource_id, Transport.Validators.MobilityDataGTFSValidator))
```

Comment:
```suggestion
    (DB.MultiValidation.resource_latest_validation(resource_id, @gtfs_validator) ||
       DB.MultiValidation.resource_latest_validation(resource_id, @mobilitydata_gtfs_validator)
```

Ceci ne fonctionne pas. Si vous avez une explication !

Reply:
mdr, je l'ai. C'est pas dans le même module. 😮‍💨

Reply:
ah ah !